.changeset/yellow-needles-love.md

```diff
@@ -0,0 +1,3 @@
+---
+"@finos/legend-query": patch
+---
```

Comment:
if we are improving the algo, i think we should include a changeset of what we are doing

Reply:
Not really improving the algo. That would be a later commit though. May be i should just change the title